libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/building/view/MapboxBuildingView.kt

```diff
@@ -0,0 +1,103 @@
+package com.mapbox.navigation.ui.maps.building.view
+
+import com.mapbox.maps.MapboxMap
+import com.mapbox.maps.QueriedFeature
+import com.mapbox.maps.Style
+import com.mapbox.maps.extension.style.expressions.dsl.generated.literal
+import com.mapbox.maps.extension.style.expressions.generated.Expression
+import com.mapbox.maps.extension.style.layers.addLayer
+import com.mapbox.maps.extension.style.layers.generated.FillExtrusionLayer
+import com.mapbox.maps.extension.style.layers.getLayerAs
+import com.mapbox.navigation.ui.maps.building.model.MapboxBuildingHighlightOptions
+
+/**
+ * Mapbox default view that adds/updates/removes building layer and building highlight
+ * for a specific feature geometry and applies appropriate properties to the building layer
+ * on top of [MapboxMap] using [MapboxBuildingHighlightOptions]
+ */
+class MapboxBuildingView {
+
+    /**
+     * Access the expression used to create the building's height.
+     */
+    private val buildingHeightExpression: Expression =
+        Expression.interpolate(
+            Expression.linear(), Expression.zoom(),
+            literal(15.0), literal(0),
+            literal(15.05), Expression.get("height")
```

Comment:
Could we refactor this magic number (`"height"`) into a `const` (within the `companion object`)?